v1/lib/input/consoleinput.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2016 CN-Consult GmbH
+ * @author Max Späth <max.spaeth@cn-consult.eu>
+ */
+
+require_once __DIR__."/../baseinput.php";
+require_once __DIR__."/../gamefield.php";
+
+/**
+ * This plugin takes input from the console to specify the width/height of the gamefield and which sets should be set alive.
+ * To use it, just set 'console' as --input parameter and set the --numCycles (Number of rounds should be played and the --output (console, png or gif).
```

Comment:
numcycles information is not necessary here, this is documentated in the runGame